message_center_compassion/models/ir_http.py

```diff
@@ -20,8 +20,10 @@
 
 try:
     from jwt import JWT, jwk
-except ImportError:
+    from jwt.exceptions import JWTDecodeError
+except ImportError as e:
     _logger.error("Please install python jwt")
+    raise e
```

Comment:
If you raise the error we won't be able to run odoo even when the module is not installed I think.

Reply:
I though that since communication with the US is a hard requirement, it should be good to let Odoo crash if the requirements are not met instead of letting it run. It's up to you :)